### PR TITLE
Update link to point to WinUI 3 docs

### PR DESCRIPTION
**Pull request by marcelwgn**
## Description

Adjusts one link as part of #1344
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -26,4 +26,10 @@
     </ItemGroup>
   </Target>
 
+  <ItemGroup>
```
Comment (karkarl) [inline Directory.Build.props:29]: This should be removed since not part of the PR :)

Comment (marcelwgn) [reply Directory.Build.props]: Good point, fixed now

**Review by karkarl — APPROVED** [review]
